tests/test_services_service.py

```diff
@@ -120,8 +116,6 @@ def get_config_json_data_v2(**kwargs: t.Any) -> t.Dict[str, t.Any]:
                     "instances": [kwargs.get("instance_0")],
                     "token": kwargs.get("token"),
                     "multisig": kwargs.get("multisig"),
-                    "staked": True,
-                    "on_chain_state": kwargs.get("on_chain_state"),
```

Comment:
Are you sure about these changes? this methods should return how a  V2  json would look like, and in these there were 'staked' and 'on_chain_state', no?

Reply:
So this code will only be executed when someone is upgrading from verrsion 0 to 6 (curernt). And since, this version is going to be backward incompatible, I see it fine to remove them. But, I can revert it back also if you'd like to keep it as it always has been, AFAICS it won't make any difference. Wdyt?